src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
Here's a link to the image with the names fixed:

https://docs.google.com/drawings/d/1mb6vv77Yz4j3xlgzxlr_wBFuck2vhEH2QnEovLAizk0/edit?usp=sharing

My question is whether object 2 gets considered when processing object 1 in frame 1.